spec/features_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe 'User Stories' do
+
+    # As a customer
+    # So that I can check if I want to order something
```

Comment:
I like the use of these user story spec. Not everyone would do that but keeps the user stories in mind.